src/commands/flinkArtifacts.ts

```diff
@@ -1,9 +1,25 @@
-import { Disposable } from "vscode";
+import vscode, { Disposable } from "vscode";
 import { registerCommandWithLogging } from ".";
 import { ContextValues, setContextValue } from "../context/values";
 import { flinkArtifactUDFViewMode } from "../emitters";
+import { FlinkArtifact } from "../models/flinkArtifact";
 import { FlinkArtifactsViewProviderMode } from "../viewProviders/multiViewDelegates/constants";
 
+/**Open a new tab set to Flink SQL type with placeholder Flink UDF registration statement for selected artifact */
+export async function queryArtifactWithFlink(selectedArtifact: FlinkArtifact | undefined) {
+  const placeholderQuery = `-- Register UDF for artifact "${selectedArtifact?.name}"
+CREATE FUNCTION \`yourFunctionNameHere\` AS 'your.class.NameHere' USING JAR 'confluent-artifact://${selectedArtifact?.id}';
+-- confirm with 'SHOW USER FUNCTIONS';
+`;
```

Comment:
Minor: what do you think about silently returning early here instead of using `?` if we get `undefined` instead of an artifact instance? The other option is offering an artifact quickpick, but that seems like overkill.

Realistically that shouldn't happen, but on the off chance it's `undefined` the document will look like this:
```sql
-- Register UDF for artifact "undefined"
CREATE FUNCTION `yourFunctionNameHere` AS 'your.class.NameHere' USING JAR 'confluent-artifact://undefined';
-- confirm with 'SHOW USER FUNCTIONS';
```